src/tests/test_utils/common_test_utils/include/common_test_utils/data_utils.hpp

```diff
@@ -194,6 +194,21 @@ void inline fill_data_random(T* pointer,
     }
 }
 
+template <class T>
+void inline fill_data_ptr_real_random_float(T* pointer,
+                                            std::size_t size,
+                                            const float min,
+                                            const float max,
+                                            const int seed) {
+    std::mt19937 gen(seed);
```

Comment:
Just minor open question: to use such generator for instead of `gtest::Random`?
@evkotov Have you checked which generator is using in `gtest::Random`?

Reply:
Minor for the future: unite two functions into one. Because float numbers could be generate by resolution using